docs/content/Getting-Started/user-quickstart-kind.md

```diff
@@ -0,0 +1,250 @@
+---
+short_name: user-quickstart-kind
+manifest_name: 'docs/content/Getting-Started/user-quickstart-kind.md'
+---
+[![User QuickStart Kind test]({{config.repo_url}}/actions/workflows/docs-ecutable-user-quickstart-kind.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-user-quickstart-kind.yml)&nbsp;&nbsp;&nbsp;
+
+<!-- 
+!!! tip "Estimated time to complete this example:" 
+    ~20 minutes (after installing prerequisites) -->
+
+## How to deploy and use <span class="Space-Bd-BT">KUBESTELLAR</span> on Kind Kubernetes Clusters
+!!! tip ""
+    === "Goals"
+        This guide will show how to:
+
+        1. quickly deploy the <span class="Space-Bd-BT">KUBESTELLAR</span> Core component on a Kind cluster using helm (ks-core), 
+        2. install the <span class="Space-Bd-BT">KUBESTELLAR</span> user commands and kubectl plugins on your computer with brew,
+        3. retrieve the <span class="Space-Bd-BT">KUBESTELLAR</span> Core component kubeconfig, 
+        4. install the <span class="Space-Bd-BT">KUBESTELLAR</span> Syncer component on two edge Kind clusters (ks-edge-cluster1 and ks-edge-cluster2), 
+        5. deploy an example kubernetes workload to both edge Kind clusters from <span class="Space-Bd-BT">KUBESTELLAR</span> Core (ks-core),
+        6. view the example kubernetes workload running on two edge Kind clusters (ks-edge-cluster1 and ks-edge-cluster2)
+        7. view the status of your deployment across both edge Kind clusters from <span class="Space-Bd-BT">KUBESTELLAR</span> Core (ks-core)
+
+        **important:** For this quickstart you will need to know how to use kubernetes' kubeconfig *context* to access multiple clusters.  You can learn more about kubeconfig context [here](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+
+!!! tip ""
+    === "Pre-reqs"
+        + [__kubectl__](https://kubernetes.io/docs/tasks/tools/) (version range expected: 1.24-1.26)
+
+        + [__helm__](https://helm.sh/docs/intro/install/) - to deploy the <span class="Space-Bd-BT">KUBESTELLAR</span>-core helm chart
+        
+        + [__brew__](https://helm.sh/docs/intro/install/) - to install the <span class="Space-Bd-BT">KUBESTELLAR</span> user commands and kubectl plugins
```

Comment:
This is pointing at the helm install instructions again.
Don't you want to point at https://docs.brew.sh/Installation ?